components/tikv_util/src/macros.rs

```diff
@@ -171,6 +171,34 @@ macro_rules! try_opt_or {
     }};
 }
 
+#[cfg(target_os = "linux")]
+#[macro_export]
+macro_rules! add_thread_name {
+    ($f:expr) => {{
+        use nix::unistd::gettid;
```

Comment:
I think you can use `$crate::nix::xxx` if nix is declared in lib.rs like `extern crate nix`.